apps/files/src/components/FilesLists/RowActionsDropdown.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <oc-drop
     v-if="displayed"
-    :boundary="`#files-file-list-action-button-small-resolution-${item.id}-active`"
+    :boundary="`#files-file-list-action-button-${item.id}-active`"
     :options="{ offset: 0 }"
```

Comment:
Since you are touching this code, mind putting `flip: false` inside the options json here? In responsive mode the dropdown can flip to the top and hide behind the top bar, see screenshot. flip: false prevents flipping to the top.

![Bildschirmfoto 2020-02-11 um 17 51 52](https://user-images.githubusercontent.com/3532843/74259191-c9882f80-4cf7-11ea-8831-2b361a02e926.png)